XSConsoleMetrics.py

```diff
@@ -14,15 +14,19 @@
 # 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 
 import xml.dom.minidom
+import sys
 
 from XSConsoleAuth import *
 from XSConsoleBases import *
 from XSConsoleLang import *
+from XSConsoleLog import *
 
-try:
-    from urllib import URLopener  # pylint: disable=no-name-in-module # for Python2
-except ImportError:
-    from urllib.request import URLopener
+if sys.version_info[0] == 2:
+    # Python 2
+    from urllib import URLopener
```

Comment:
Why not adding a
```python
   def urlopen(url):
       return URLopener().open(url)
```
here and just use `urlopen` below?

Reply:
Thanks for the suggestion!